.github/workflows/test_turnkey.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Set up Miniconda with 64-bit Python
-        uses: conda-incubator/setup-miniconda@v2
+        uses: conda-incubator/setup-miniconda@v3
```

Comment:
Any particular reason we are upgrading this?
There is nothing wrong with it, I'm just curious.

Reply:
There was some crazy livesite last week: https://github.com/conda-incubator/setup-miniconda/issues/358

Ultimately, I'm not sure if this upgrade helped, but I did it as a precaution and it definitely didn't hurt.